[2025-03-19T07:07:09.467+0000] {taskinstance.py:1149} INFO - Dependencies not met for <TaskInstance: etl_pipeline.extract scheduled__2025-03-19T06:00:00+00:00 [failed]>, dependency 'Task Instance State' FAILED: Task is in the 'failed' state.
[2025-03-19T07:07:09.472+0000] {local_task_job_runner.py:154} INFO - Task is not able to be run
